Events: event data must conform to data protection and access controls.

dehora opened the pull request:
Adds a guideline to ensure that sensitive data is not published
without first checking it is ok to do so.

tfrauenstein commented on events/events.md:
```diff
@@ -22,6 +22,12 @@ An EventType is registered with Nakadi via its _Schema Registry API_. Once the E
+Sensitive data, such as (e-mail addresses, phone numbers, etc) are subject to strict access and data protection controls. As a result, event type owners **must** ensure data is being published is in accordance with any such criteria. 
+
+Also, please note that the Nakadi API currently has no support in its OAuth scopes for restricting access to event types. It is intended in the future to allow per topic scope controls, but in the meantime be aware that event data in Nakadi is broadly accessible to any consumers with a read scope.
+
```
Title: Events must not provide sensible customer personal data.

We should more clearly say: Analog to API access permissions, there will be Event Type access permissions passed via OAuth token supported in near future by Nakadi access authorisation.

This is a temporary rule which will be obsolete as soon as Nakadi supports event type authorization: 

Sensitive personal data, such as (e-mail addresses, phone numbers, etc) are subject to strict access and data protection controls. Event type owners **must** ensure not to publish this sensible customer data.

valgog replied:
+1

fmueller commented:
This topics looks quite Zalando specific and Nakadai specific. I'm not sure if APi guidelines is the right place for this.

Furthermore, it's problematic to forbid sensible data when Nakadi is used for service-to-service communication. Also I think at the moment it's a tooling problem: we miss access scopes in events, right?

dehora commented:
>  I'm not sure if APi guidelines is the right place for this.

It is Zalando specific; that's true, but that's not unique in the guidelines. 

In any case, I've updated to match @tfrauenstein's feedback in d4d6481; just moved some sentences around, so imo it's  ready to merge. And fwiw I'm +0 on whether it goes here or somewhere else; my only observation is that this is one place teams are likely to see it, as well as API reviewers who can query event data as needed. 

/ @fmueller @valgog @whiskeysierra

dehora commented:
@fmueller @valgog @whiskeysierra @tfrauenstein small edits in 8ecbc69

fmueller commented:
👍

whiskeysierra commented:
:+1:
